ngraph/src/ngraph/op/scatter_elements_update.cpp

```diff
@@ -249,10 +249,18 @@ namespace
 
         switch (out->get_element_type())
         {
+            TYPE_CASE(i8)(arg0, arg1, arg2, arg3, out, normalized_axis);
```

Comment:
@ilyachur don't you think that this types are needed?

Reply:
@chenhu-wang Why do you add these types? 

We didn't face with these data types for this operation.

Reply:
I am implementing according [ops spec](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/movement/ScatterElementsUpdate_3.md). It says input is data type of T, and T is any numeric type. So I added this support and test. Do you think It would be possible data type need support in the future? for example, If some onnx model contain [scatterElementsUpdate layer](https://github.com/onnx/onnx/blob/rel-1.7.0/docs/Operators.md#type-constraints-120), which have i8/i16/u8/u16 as input data type, we can support it directly.

Reply:
@GlebKazantaev,  do you agree with this replay for your request?
https://github.com/openvinotoolkit/openvino/pull/909#discussion_r450228840
https://github.com/openvinotoolkit/openvino/pull/909#discussion_r450251654

Reply:
@GlebKazantaev ,
> @ilyachur was working on cleaning up some reference implementations to reduce the nGraph library binary size. The probability that we will have i8 and I16 as input is very low, so do not extend support for them.



> @lazarevevgeny @ilyachur Thank you for clarification. Have moved extension and test for these data type!